vald/sign.go

```diff
@@ -119,9 +118,9 @@ func GetSignCommand() *cobra.Command {
 				signDetails := map[string]string{
```

Comment:
is signDetails purely used for console output?

Reply:
validator will save the output to a file, and signatures will be aggregated across all to submit the emergency batch